Concision edits to what-is-batch.d

landermkerbey opened the pull request:
Altered the opening paragraphs of "What Is AWS Batch?" to remove some obtrusive repetition, as well as generally tighten up the wording.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

landermkerbey committed:
Concision edits to what-is-batch.d

Altered the opening paragraphs of "What Is AWS Batch?" to remove some obtrusive repetition, as well as generally tighten up the wording.

nrdlngr commented on doc_source/what-is-batch.md:
```diff
@@ -1,8 +1,8 @@
+AWS Batch enables you to run batch computing workloads on the AWS Cloud\. Batch computing is a common way for developers, scientists, and engineers to access large amounts of compute resources, and AWS Batch removes the undifferentiated heavy lifting of configuring and managing the required infrastructure, similar to traditional batch computing software\. This service can efficiently provision re ...
 
-As a fully managed service, AWS Batch enables developers, scientists, and engineers to run batch computing workloads of any scale\. AWS Batch automatically provisions compute resources and optimizes the workload distribution based on the quantity and scale of the workloads\. With AWS Batch, there is no need to install or manage batch computing software, which allows you to focus on analyzing resu ...
+As a fully managed service, AWS Batch scales to any task, automatically provisioning compute resources and optimizing workload distribution based on the quantity and scale of the workloads\. AWS Batch bypasses the installation and management of batch computing software, saving time and reducing operational complexity so you can focus on analyzing results and solving problems\.
```
I agree that the 'scientists, developers, and engineers' phrase gets repetitive, but I'm wary of the "scales to any task" and "bypasses the installation and management..." suggestions. They imply that AWS Batch is an actor that is doing something instead of a service that enables customers to do work.

Can you focus more on just reducing the repetition that you identified, and then I can probably merge this.

landermkerbey committed:
Update what-is-batch.md

Second pass to fix for issues of implied agency of AWS Batch, refocus on customer usage.

landermkerbey commented:
What do you think of this version? I tried to put the focus back on Batch as a tool for users while still trimming the redundancy. Let me know if it works.

nrdlngr commented:
Sorry I let this sit idle for so long. I think I'm going to tweak this a little after merging it, but your suggestion to reduce redundancy is valid, so I want you to get credit for that.
